bika/lims/browser/worksheet/templates/add_analyses.pt

```diff
@@ -1,133 +1,137 @@
-						tal:content="python:services_list.getValue(service)"/>
-				</tal:options>
-			</select>
+          <div class="field">
+            <label tal:attributes="for string:${form_id}_FilterByService"
+                   i18n:translate="">Service</label>
+            <select id="ServiceSelector"
+                    tal:attributes="name string:${form_id}_FilterByService"
+                    tal:define="services_list view/getServices"
+                    class="listing-filter">
+              <option value="any" i18n:translate="">Any</option>
+              <tal:options repeat="service services_list">
+                <option
+                  tal:attributes="value python:service;
+                                  selected python: view.request.form.get(form_id+'_Title', '') == service and 'selected' or ''"
+                  tal:content="python:services_list.getValue(service)"/>
+              </tal:options>
+            </select>
+          </div>
 
-			<label tal:attributes="for string:${form_id}_FilterByClient"
-				i18n:translate="">Client</label>
-			<select id="ClientSelector"
-					tal:attributes="name string:${form_id}_FilterByClient"
-					tal:define="clients_list view/getClients"
-					class="listing-filter">
-				<option value="any" i18n:translate="">Any</option>
-				<tal:options repeat="client clients_list">
-					<option
-						tal:attributes="
-							value python:client;
-							selected python: view.request.form.get(form_id+'_getClientTitle', '') == client and 'selected' or ''"
-						tal:content="python:clients_list.getValue(client)"/>
-				</tal:options>
-			</select>&nbsp;
-			<a tal:attributes="href view/view_url" i18n:translate=""
-				class="ws-analyses-search-button">Search</a>
-		</fieldset>
+          <div class="field">
+            <label tal:attributes="for string:${form_id}_FilterByClient"
+                   i18n:translate="">Client</label>
+            <select id="ClientSelector"
+                    tal:attributes="name string:${form_id}_FilterByClient"
+                    tal:define="clients_list view/getClients"
+                    class="listing-filter">
+              <option value="any" i18n:translate="">Any</option>
+              <tal:options repeat="client clients_list">
+                <option
+                  tal:attributes="value python:client;
+                                  selected python: view.request.form.get(form_id+'_getClientTitle', '') == client and 'selected' or ''"
+                  tal:content="python:clients_list.getValue(client)"/>
+              </tal:options>
+            </select>
+          </div>
 
-	</tal:searchform>
-	<fieldset/>
+          <!-- Search Button -->
+          <a tal:attributes="href view/view_url"
+             class="ws-analyses-search-button"
+             i18n:translate="">Search</a>
 
-	<div id="folderlisting-main-table"
-		tal:content="structure view/contents_table"/>
+        </fieldset>
+      </tal:searchform>
```

Comment:
Before this PR, the filter elements were appearing in-line, now appear stacked. Probably because you wrapped the filter elements inside `<div class="field">..</div>` 's. 

I think it would be much better if we could keep the "traditional "inline" layout while assuring it gets rendered correctly when `senaite.lims` UI is installed. I suggest to include all elements inside a `<div class='row'>` and wrap each of them by a `<div class='col-sm-4 field'>` and add a css rule in `ploneCustom.css` similar to:
```css
.template-add_analyses.portaltype-worksheet .criteria .field {
    clear: none;
    float: left;
    padding-right: 20px;
}
```

Reply:
perfect, will change that.

Since `ploneCustom.css` is not active in `senaite.lims`: https://github.com/senaite/senaite.lims/blob/master/src/senaite/lims/profiles/default/cssregistry.xml#L66

we can use in general all the bootstrap classes and divs and change it for the core just in `ploneCustom.css`, that's cool!